source/installation-guide/wazuh-indexer/step-by-step.rst

```diff
@@ -184,37 +186,44 @@ Cluster initialization involves configuring a default ISM policy, loading new ce
 Testing the cluster installation
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-#. Replace ``<WAZUH_INDEXER_IP_ADDRESS>`` and run the following commands to confirm that the installation is successful.
+#. Replace ``<INDEXER_USERNAME>``, ``<INDEXER_PASSWORD>`` and ``<WAZUH_INDEXER_IP_ADDRESS>`` with the Wazuh indexer username and password that you set using the Keystore tool commands:
 
-   .. code-block:: console
+  .. code-block:: console
+
+      # /var/ossec/bin/wazuh-keystore -f indexer -k username -v <INDEXER_USERNAME>
+      # /var/ossec/bin/wazuh-keystore -f indexer -k password -v <INDEXER_PASSWORD>
+         
+  And run the following commands to confirm that the installation is successful.
+
+  .. code-block:: console
 
-      # curl -k -u admin:admin https://<WAZUH_INDEXER_IP_ADRESS>:9200
-
-   .. code-block:: none
-      :class: output accordion-output
-
-      {
-        "name" : "node-1",
-        "cluster_name" : "wazuh-cluster",
-        "cluster_uuid" : "095jEW-oRJSFKLz5wmo5PA",
-        "version" : {
-          "number" : "7.10.2",
-          "build_type" : "rpm",
-          "build_hash" : "db90a415ff2fd428b4f7b3f800a51dc229287cb4",
-          "build_date" : "2023-06-03T06:24:25.112415503Z",
-          "build_snapshot" : false,
-          "lucene_version" : "9.6.0",
-          "minimum_wire_compatibility_version" : "7.10.0",
-          "minimum_index_compatibility_version" : "7.0.0"
-        },
-        "tagline" : "The OpenSearch Project: https://opensearch.org/"
-      }
-
-#. Replace ``<WAZUH_INDEXER_IP_ADDRESS>`` and run the following command to check if the single-node or multi-node cluster is working correctly. 
+      # curl -k -u <INDEXER_USERNAME>:<INDEXER_PASSWORD> https://<WAZUH_INDEXER_IP_ADRESS>:9200
+
+  .. code-block:: none
+    :class: output accordion-output
+
+    {
+      "name" : "node-1",
+      "cluster_name" : "wazuh-cluster",
+      "cluster_uuid" : "095jEW-oRJSFKLz5wmo5PA",
+      "version" : {
+        "number" : "7.10.2",
+        "build_type" : "rpm",
+        "build_hash" : "db90a415ff2fd428b4f7b3f800a51dc229287cb4",
+        "build_date" : "2023-06-03T06:24:25.112415503Z",
+        "build_snapshot" : false,
+        "lucene_version" : "9.6.0",
+        "minimum_wire_compatibility_version" : "7.10.0",
+        "minimum_index_compatibility_version" : "7.0.0"
+      },
+      "tagline" : "The OpenSearch Project: https://opensearch.org/"
+    }
+
+#. Replace ``<INDEXER_USERNAME>``, ``<INDEXER_PASSWORD>`` and ``<WAZUH_INDEXER_IP_ADDRESS>`` and run the following command to check if the single-node or multi-node cluster is working correctly. 
   
-   .. code-block:: console
+  .. code-block:: console
```

Comment:
```suggestion
    .. code-block:: console
```

Reply:
Done in [Fix new numbering problems](https://github.com/wazuh/wazuh-documentation/pull/7000/commits/641e5635dfba6d4fa5bb6ef08cba14cfbd40fc82) commit